README.md

```diff
@@ -1,2 +1,23 @@
 # flask-bucketlist
-A Flask API for a bucket list service
+A Flask API for a bucket list service.
```

Comment:
Consider adding badges to your repo to show how your project fairs on travis, coveralls e.t.c..